src/pineko/scaffold.py

```diff
@@ -0,0 +1,59 @@
+"""Tools related to generation and managing of a pineko project."""
+
+import os
+import pathlib
+
+from .configs import needed_keys
+
+
+def set_up_project(configs):
+    """Set up all the folders spelled up in the configs dictionary.
+
+    Parameters
+    ----------
+    configs : dict
+        configs dictionary containing all the paths to be setted up
+    """
+    for path in configs["paths"]:
+        if path == "root":
+            continue
+        if isinstance(configs["paths"][path], pathlib.Path):
+            if configs["paths"][path].suffix == ".yaml":
+                continue
+            os.makedirs(configs["paths"][path], exist_ok=True)
+        elif isinstance(configs["paths"][path], dict):
+            for log_path in configs["paths"][path]:
+                if isinstance(configs["paths"][path][log_path], pathlib.Path):
+                    os.makedirs(configs["paths"][path][log_path], exist_ok=True)
+                else:
+                    raise TypeError(f"Not recognized entry {log_path} in configs")
+        else:
+            raise TypeError(f"Not recognized entry {path} in configs")
+
+
+def check_folders(configs):
+    """Check if all the folders spelled up in configs exist.
+
+    Parameters
+    ----------
+    configs : dict
+        configs dictionary containing all the paths to be checked
+    """
+    for key in needed_keys:
+        if key not in configs["paths"]:
+            raise ValueError(f"{key} not found in config file")
+        if key == "operator_card_template":
```

Comment:
You mean if I change `operator_card_template` to `operator_card_template_name`? Because in any case I would need the exception: `operator_card_template_name` would be something like `_template.yaml` and I don't want to check if it exists because I can't, it will be stored inside a theory folder